src/components/Trade/hooks/useSwapper/utils.ts

```diff
@@ -53,27 +52,14 @@ export const isSupportedNonUtxoSwappingChain = (
 }
 
 // Pure functions
-export const getFirstReceiveAddress: GetFirstReceiveAddress = async ({
-  accountSpecifiersList,
-  buyAsset,
+export const getSelectedReceiveAddress: GetFirstReceiveAddress = async ({
```

Comment:
Made this generic - get the receive address of whatever `AccountId` we pass in, as we have those in form state now.

Reply:
nitpick - rename type too